.github/workflows/promptflow-sdk-cli-azure-test.yml

```diff
@@ -1,28 +1,45 @@
-name: promptflow-sdk-cli-azure-e2e-test
+# execute tests in src/promptflow/tests/sdk_cli_azure_test
+# when pull_request triggers, use replay test mode
+# otherwise (schedule/workflow_disptach), use live test mode
```

Comment:
shall user also be able to use replay mode when dispatch ?

Reply:
offline synced, anyone who is able to dispatch trigger, should be OK to use live mode